README.adoc

```diff
@@ -91,12 +91,39 @@ The implementation follows the standard build/test/archive pattern.
 The method targets compatibility with link:https://github.com/jenkinsci/gradle-jpi-plugin[Gradle JPI Plugin],
 and it may not work for other use-cases.
 
-The interface is similar to `buildPlugin()`, but some features are not supported.
+==== Optional arguments
+
+* `repo` (default: `null`  inherit from Multibranch) - custom Git repository to check out
+* `failFast` (default: `true`) - instruct the build to fail fast when one of the configurations fail
+* `platforms` (default: `['linux', 'windows']`) - Labels matching platforms to
+  execute the steps against in parallel
+* `jdkVersions` (default: `[8]`) - JDK version numbers, must match a version
+    number jdk tool installed
+* `configurations`: An alternative way to specify `platforms`, `jdkVersions` (that can not be combined
+  with any of them)
+** Those options will run the build for all combinations of their values. While that is desirable in
+  many cases, `configurations` permit to provide a specific combinations of label and java/jenkins versions to use
+** It is also possible to use a  `buildPlugin.recommendedConfigurations()` method to get recommended configurations for testing. 
+Note that the recommended configuration may change over time, 
+and hence your CI may break if the new recommended configuration is not compatible
```

Comment:
Shouldn't there be a dot at the end of the sentence?

Reply:
I usually do not put dots in lists, but no strong opinion